pgUI/pgUI/stories/Button.stories.ts

```diff
@@ -0,0 +1,23 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+// import { Button } from './Button';
+import Button from '../components/Button';
```

Comment:
Not a very sophisticated example but here is a simple button. We can start adding components that are more advanced like the InstanceCard or even a certain extension UI view as we go.